### PR TITLE
(Feature) Planner and discoverer blueprints for Brooklyn

### PR DESCRIPTION
**Pull request by szenzaro**
Added blueprints and readme for planner and discoverer

### CODE REVIEW

**Comment by andreaturli** [comment]
Thx @szenzaro 
Given these updates, do we need to update also 'seaclouds-on-byon' and 'seaclouds' blueprints?

**Comment by szenzaro** [comment]
@andreaturli, yes I updated them. Could you please check their correctness?

I detached the planner as a child from the Dashboard since the planner is an independent service.

File: usage/installer/src/main/resources/eu.seaclouds.planner/planner_blueprint.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  shell.env:
+    PLANNER_PORT: $brooklyn:config("planner.config.port")
+    DISCOVERER_URL: $brooklyn:config("planner.config.discoverer.url")
+    OPTIMIZER_URL: $brooklyn:config("planner.config.optimizer.url")
+    MONITOR_GENERATOR_URL: $brooklyn:config("planner.config.monitor_generator.url")
+    SLA_GENERATOR_URL: $brooklyn:config("planner.config.sla_generator.url")
+    PLANNER_SERVICE_URL: $brooklyn:config("planner.config.service.url")
+
+  launch.command: |
```
Comment (rosogon) [inline usage/installer/src/main/resources/eu.seaclouds.planner/planner_blueprint.yml]: Is not there a simpler way for executing a jar?

Comment (szenzaro) [inline usage/installer/src/main/resources/eu.seaclouds.planner/planner_blueprint.yml]: @andreaturli helped me to write this. If there is a simpler way to write it he would probably know better than me. Andrea, should we change this one?

Comment (rosogon) [inline usage/installer/src/main/resources/eu.seaclouds.planner/planner_blueprint.yml]: If Andrea helped you, then it should be the way. So, no need to change. I simply thought it was going to be easier.

**Comment by rosogon** [comment]
@szenzaro . This branch has conflicts. You should fix them before it can be merged.

File: usage/installer/src/main/resources/eu.seaclouds.planner/planner_blueprint.yml (hunk 1 above)
Comment (szenzaro) [inline usage/installer/src/main/resources/eu.seaclouds.planner/planner_blueprint.yml]: I am not an expert of Brooklyn blueprints, so I really do not know if there is a better way to write it.

**Comment by rosogon** [comment]
seaclouds-on-byon gives the following error:  Invalid YAML: null; mapping values are not allowed here; in 'reader', line 79, column 11: name: Discoverer ^

**Comment by szenzaro** [comment]
I removed the seaclouds on byon and seaclouds.yaml.

**Comment by rosogon** [comment]
@szenzaro : Can we close this PR?

**Comment by szenzaro** [comment]
I think this one should be merged after the #105. If there are other reasons why we should close this, let me know.

**Comment by rosogon** [comment]
Hi @szenzaro . #109 changes location of blueprints, and already has the blueprint for the planner. Maybe @andreaturli can say if this PR is needed or not.

**Comment by andreaturli** [comment]
if we merge https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/109 (which is *highly* recommended!!) we won't need these single yaml files anymore.

**Comment by szenzaro** [comment]
Ok so we can close this PR. But we need to add the discoverer blueprint as well.

**Comment by andreaturli** [comment]
please see https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/109, it now contains also the discoverer. Tested already on BYON

**Pull request closed without merging**